admin_guide/managing_projects.adoc

```diff
@@ -356,12 +357,14 @@ with this label will not have a maximum of project requests.
 to any user that doesn't satisfy the previous two rules. Because rules are evaluated
 in order, this rule should be specified last.
 
+ifdef::openshift-enterprise,openshift-origin[]
 [NOTE]
 ====
 xref:../admin_guide/manage_users.adoc#managing-users-managing-user-and-group-labels[Managing
 User and Group Labels] provides further guidance on how to add, remove, or show
 labels for users and groups.
 ====
+ifdef::openshift-enterprise,openshift-origin[]
```

Comment:
Do you need this one? (I don't see an endif for the last one.)

Reply:
Good catch. The last one should have been an endif. However, as of today, the topic linked to in that note box is added to Dedicated, so no reason for the ifdef statements anymore: https://github.com/openshift/openshift-docs/pull/10498